guides/common/modules/con_managing-red-hat-subscriptions.adoc

```diff
@@ -1,40 +1,71 @@
 [id="Managing_Red_Hat_Subscriptions_{context}"]
 = Managing Red Hat subscriptions
 
-{ProjectName} can import content from the Red{nbsp}Hat Content Delivery Network (CDN).
-{Project} requires a Red{nbsp}Hat subscription manifest to find, access, and download content from the corresponding repositories.
-You must have a Red{nbsp}Hat subscription manifest containing a subscription allocation for each organization on {ProjectServer}.
-All subscription information is available in your Red Hat Customer Portal account.
+{ProjectName} can import content from the {Team} Content Delivery Network (CDN).
+{Project} requires a {Team} subscription manifest to find, access, and download content from the corresponding repositories.
+You must have a {Team} subscription manifest containing a subscription allocation for each organization on {ProjectServer}.
+All subscription information is available in your {Team} Customer Portal account.
```

Comment:
This text is available in upstream docs as well. Please, don't replace `Red{nbsp}Hat` with the `{Team}` attribute, it has to be hard coded, otherwise it will result in nonsensical concepts, such as "Foreman community CDN", which doesn't exist.